.github/workflows/lint-and-test-management.yml

```
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  pull_request:
+    paths:
+      - "management/**"
+jobs:
+  management-linter:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2.3.4
+    
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+
+    - uses: actions/cache@v2.1.5
+      id: yarn-cache
+      with:
+        path: |
+          ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          **/management/node_modules
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+
+    - name: dependencies
+      if: steps.yarn-cache.outputs.cache-hit != 'true'
+      run: cd management/ && yarn install
+
+    - name: linter
+      run: cd management/ && yarn lint
+
+  management-test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2.3.4
+    
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+
+    - uses: actions/cache@v2.1.5
+      id: yarn-cache
+      with:
+        path: |
+          ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          **/management/node_modules
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+    
+    - name: dependencies
+      if: steps.yarn-cache.outputs.cache-hit != 'true'
+      run: cd management/ && yarn install
```

```suggestion
    - name: dependencies
      if: steps.yarn-cache.outputs.cache-hit != 'true'
      run: yarn install
      working-directory: management
```

source: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun